iphone/Maps/UI/BottomMenu/TabBar/BottomTabBarViewController.swift

```diff
@@ -1,3 +1,6 @@
+
+private let kUDDidShowHelpButtonEducationalHint = "kUDDidShowHelpButtonEducationalHint"
```

Comment:
Let's name it something like "first time routing educational hiht". We likely provide other hints too for the help button.

Reply:
Fixed!